src/svg/timer.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="120" height="120" viewBox="0 0 120 120" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We are trying to have icons on a 24x24 grid and 0 0 24 24 viewbox. Could you ask to have the icon in this size?